mods/authentication/src/utils/jwt.ts

```diff
@@ -0,0 +1,71 @@
+import path from 'path'
+import { readFile } from 'fs'
+import { promisify } from 'util'
+import { sign, verify } from 'jsonwebtoken'
+import logger from '@fonos/logger'
+import JwtPayload from '../utils/jwtPayload'
+import ITokenManager from './ITokenManager'
+/*
+ * issuer 		— Module who issues the token.
```

Comment:
In this context, the issuer is the organization instead of a module. For example, the issuer could be `acme.com`.
Consider updating this comment to `Organization issue the token`